ilc/server/tailor/fetch-template.spec.js

```diff
@@ -0,0 +1,64 @@
+const chai = require('chai');
+const sinon = require('sinon');
+
+const fetchTemplates = require('./fetch-template');
+
+describe('fetch templates', () => {
+  const configsInjector = {
+    inject: sinon.stub(),
+  };
+
+  const newrelic = {
+    setTransactionName: sinon.spy(),
+  };
+
+  const registryService = {
+    getTemplate: (arg) => arg,
+  };
+
+  const request = {
+    router: {},
+    ilcState: 'ilcState text'
+  };
+
+  const parseTemplate = sinon.stub();
```

Comment:
You need to reset all shared stubs/spies after every test to ensure that tests don't impact each other